src/Compilers/VisualBasic/Portable/Binding/Binder_Invocation.vb

```diff
@@ -3009,6 +3014,8 @@ ProduceBoundNode:
                                 End If
 
                                 argumentNamesLocationsBuilder.Add(id.GetLocation())
+                            ElseIf argumentNamesBuilder IsNot Nothing Then
+                                argumentNamesBuilder.Add(Nothing)
```

Comment:
I'll add some tests explicitly checking the locations of arguments.

---
In reply to: [122315921](https://github.com/dotnet/roslyn/pull/20182#discussion_r122315921) [](ancestors = 122315921)